### PR TITLE
Add catalog info for backstage

### PR DESCRIPTION
**Pull request by cdvultur**
Adding catalog infor for backstage.

### CODE REVIEW

File: catalog-info.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  tags:
+    - go
+    - ginkgo
+    - test-framework
+    - product-quality
+    - openshift
+  namespace: quality-community
+  annotations:
+    github.com/project-slug: openshift-kni/eco-goinfra
+    backstage.io/techdocs-ref: https://github.com/openshift-kni/eco-goinfra/wiki
```
Comment (trewest) [inline catalog-info.yaml:20]: Should we have something here?

Comment (cdvultur) [reply catalog-info.yaml:20]: From what I see in other examples we should point to the documentation. 

Maybe it is better to have it point to the README ?

File: catalog-info.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Catalog entry for Backstage [backstage.io]
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: eco-goinfra
+  title: eco-goinfra
+  description: Ecosystem QE framework
+  links:
+    - title : Ginkgo
```
Comment (kononovn) [inline catalog-info.yaml:9]: This set of tools doesn't use ginkgo

File: catalog-info.yaml (hunk 1 above)
Comment (trewest) [reply catalog-info.yaml:20]: Yeah I think the readme makes sense

Comment (cdvultur) [reply catalog-info.yaml]: Updated to use README.

File: catalog-info.yaml (hunk 2 above)
Comment (cdvultur) [reply catalog-info.yaml]: Correct! Updated it to go.dev, so that it is in sync with Readme

**Review by trewest — APPROVED** [review]
/lgtm

**Review by kononovn — CHANGES_REQUESTED** [review]
1 comment

**Comment by cdvultur** [comment]
Updated. Thanks !

File: catalog-info.yaml [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# Catalog entry for Backstage [backstage.io]
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: eco-goinfra
+  title: eco-goinfra
+  description: Ecosystem QE framework
```
[the remark this answers is not in the document]
Comment (cdvultur) [reply catalog-info.yaml]: Updated it.

Thanks

**Review by kononovn — APPROVED** [review]
/lgtm

**Pull request merged**